subprojects/language-jvm/src/testFixtures/groovy/org/gradle/api/tasks/compile/AbstractCompilerContinuousIntegrationTest.groovy

```diff
@@ -24,10 +24,6 @@ abstract class AbstractCompilerContinuousIntegrationTest extends AbstractContinu
         executer.withWorkerDaemonsExpirationDisabled()
     }
 
-    def cleanup() {
-        gradle.cancel()
```

Comment:
Why is cancel not needed anymore or why it was needed before?

Reply:
The base class already does a null-safe cancel, no need to repeat this in the sub-classes.

Reply:
See here: https://github.com/gradle/gradle/blob/cae38fa7cb471fa27cfe96998fad8ec341a37c89/subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/AbstractContinuousIntegrationTest.groovy#L271-L293